### ISSUE TITLE
No Pipfile found. Type 'pipenv' for help creating a new project.

### ISSUE THREAD

**Issue by mirosval** [kennethreitz/pipenv#301]
When I type `pipenv` I get this instead of the help that this very line promises. This is on OSX Pipenv 3.5.5.

```shell
~/work/tests/nlp-client » pip install -U pipenv                                                                                                                                                                                                                                                                             
Requirement already up-to-date: pipenv in /usr/local/lib/python2.7/site-packages
Requirement already up-to-date: pip in /usr/local/lib/python2.7/site-packages (from pipenv)
Requirement already up-to-date: virtualenv in /usr/local/lib/python2.7/site-packages (from pipenv)
Requirement already up-to-date: pew>=0.1.26 in /usr/local/lib/python2.7/site-packages (from pipenv)
Requirement already up-to-date: virtualenv-clone>=0.2.5 in /usr/local/lib/python2.7/site-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: pythonz-bd>=1.10.2 in /usr/local/lib/python2.7/site-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: pathlib; python_version == "2.7" in /usr/local/lib/python2.7/site-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: backports.shutil-get-terminal-size; python_version == "2.7" in /usr/local/lib/python2.7/site-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: shutilwhich; python_version == "2.7" in /usr/local/lib/python2.7/site-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: setuptools>=17.1 in /usr/local/lib/python2.7/site-packages (from pew>=0.1.26->pipenv)
Requirement already up-to-date: six>=1.6.0 in /usr/local/lib/python2.7/site-packages (from setuptools>=17.1->pew>=0.1.26->pipenv)
Requirement already up-to-date: packaging>=16.8 in /usr/local/lib/python2.7/site-packages (from setuptools>=17.1->pew>=0.1.26->pipenv)
Requirement already up-to-date: appdirs>=1.4.0 in /usr/local/lib/python2.7/site-packages (from setuptools>=17.1->pew>=0.1.26->pipenv)
Requirement already up-to-date: pyparsing in /usr/local/lib/python2.7/site-packages (from packaging>=16.8->setuptools>=17.1->pew>=0.1.26->pipenv)
~/work/tests/nlp-client » pipenv                                                                                                                                                                                                                                                                                            
No Pipfile found. Type 'pipenv' for help creating a new project.
~/work/tests/nlp-client »
```

**Comment by nateprewitt** [comment]
Hey @mirosval, thanks for letting us know about this. This is an issue introduced in 3.5.5 because of poor testing before we merged #284. I'll look into getting this cleaned up and released today.

**Comment by nateprewitt** [comment]
This should be resolved in 3.5.6 with 51a80b0
